src/OrderSourceReader.cs

```diff
@@ -215,15 +213,6 @@ public static void UpdateExportedOrdersInDb(string orderStateIDAfterExport, SqlC
                 }
             }
         }
-
-        private static void ClearOrderCache(IEnumerable<string> orderIds)
```

Comment:
that needs to be restored back as it is not related to the scope of this task, it clears only the cache for the orders that were exported, and its needed as there was a bug related to that problem so that code can not be removed

Reply:
ok i've re-implemented it